src/utils.ts

```diff
@@ -1,5 +1,5 @@
 import { lstatSync, accessSync, constants, readFileSync } from "node:fs";
-import { createHash } from "node:crypto";
+import { createHash, getFips } from "node:crypto";
```

Comment:
Since there is chance that export might not exists we should use default export `import nodeCrypto from ...`

then when checking, we can use `nodeCrypto.getFips?();`

Reply:
`getFips` exists in Deno but throws a `Not implemented` error at runtime.

Using `nodeCrypto.getFips?.()` is indeed safer.

Update.

Reply:
Perhaps we can have a `hasFips` util function that uses try catch and assume `0` if fails? Having hardoced deno check might introduce issues later if deno supports fips.

Reply:
Great idea 👍, updated